docs/2.8.0/docs/daml-finance/instruments/structured-product.rst

```diff
@@ -0,0 +1,78 @@
+.. Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+How To Use the Structured Product Instrument Package
```

Comment:
I we agreed that this should be singular, we should be consistent and do it across all instrument pages